cavityDisplayGUI.py

```diff
@@ -1,13 +1,14 @@
+import sys
 from dataclasses import dataclass
+from functools import partial
+from typing import List
 
-import sys
 from PyQt5.QtGui import QColor
 from PyQt5.QtWidgets import QHBoxLayout, QWidget
 from epics import PV
-from functools import partial
 from pydm import Display
-from pydm.widgets import PyDMEmbeddedDisplay, PyDMRelatedDisplayButton, PyDMTemplateRepeater
-from typing import List
+from pydm.widgets import PyDMEmbeddedDisplay, PyDMRelatedDisplayButton, PyDMTemplateRepeater, \
+    PyDMDrawingLine
```

Comment:
You can avoid the "\" if you just put all of them in parentheses

Reply:
Thank you for pointing that out! Fixed